docs/components/breadcrumbs.md

```diff
@@ -148,6 +148,50 @@ Content to be displayed between breadcrumb items, defaults to a chevron.
 </KBreadcrumbs>
 ```
 
+### breadcrumb-icon
```

Comment:
It already is that except you switched whether we lead with `icon` or end with it? I am following the approach we used for the item icon slot in `KTreelist`. I don't understand what you're asking for.

Reply:
I can switch `icon` to lead, but like I said it already is using just `icon-{key}`. The only time `breadcrumb` is thrown in is when there is no key and we are using the index, which respectfully I think `icon-breadcrumb-1` is more intuitive than `icon-1`, so I will keep.

Reply:
It isn't POLA because it is explained in the docs. We haven't ever made `key` required previously and you aren't required to use the icon slot so it doesn't make sense to make `key` required now.